src/Backend/Modules/MediaGalleries/Domain/MediaGallery/Command/UpdateMediaGalleryHandler.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Backend\Modules\MediaGalleries\Domain\MediaGallery\Command;
+
+use Backend\Modules\MediaGalleries\Domain\MediaGallery\MediaGallery;
+
+final class UpdateMediaGalleryHandler
+{
+    /**
+     * @param UpdateMediaGallery $updateMediaGallery
+     */
+    public function handle(UpdateMediaGallery $updateMediaGallery)
+    {
+        /** @var MediaGallery $mediaGallery */
+        $mediaGallery = MediaGallery::fromDataTransferObject($updateMediaGallery);
```

Comment:
since you don't reuse it I would pass it to the method without creating a parameter